### PR TITLE
Feat: readable dates

### PR DESCRIPTION
**Pull request by dleard**
- Change UTC format dates to readable format

### CODE REVIEW

**Review by kriscooke — COMMENTED** [review]
I think we should put off a decision on this until Raj returns next week (see comment in [GGIRCS-1620](https://youtrack.button.is/issue/GGIRCS-1620)

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]
Can we have a single file with `dateFormat` and `dateTimeFormat` functions in it? 
This is the best way to ensure consistency across the application, instead of copy/pasting the time zone and format strings.
`moment.tz` probably shouldn't be used directly by other files unless there's some date comparison needed (e.g in `ReportingYearTable` or `KeyDates`)

**Comment by kriscooke** [comment]
Agreed, and in that file we may need a couple of possible date formats (eg: date only, date with timestamp etc.)

**Comment by dleard** [comment]
@kriscooke @matthieu-foucault I removed the moment imports and TIME_ZONE/DATE_FORMAT variables from all files but AltDateInput.tsx & replaced them with calls to the functions in formatDates.ts

File: app/functions/formatDates.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import moment from 'moment-timezone';
+
+const TIMEZONE = 'America/Vancouver';
+const FORMAT_TYPE = {
+  seconds: 'MMM D, YYYY hh:mm:ss A (z)',
+  minutes: 'MMM D, YYYY hh:mm A (z)',
+  hours: 'MMM D, YYYY hh A (z)',
+  days_numbered: 'DD-MM-YYYY',
+  days_string: 'MMM Do, YYYY',
+  database_timestamp: 'YYYY-MM-DDTHH:mm:ss'
```
Comment (kriscooke) [inline app/functions/formatDates.ts:10]: It doesn't look like `database_timestamp` and `hours` formats are currently used anywhere - maybe we should just start with the minimum required number of formats and add any others as they become necessary?

**Review by kriscooke — APPROVED** [review]

**Pull request merged**